packages/fxa-payments-server/src/styles/tailwind.css

```diff
@@ -136,3 +136,36 @@
 .payment-button {
   @apply cta-base-p flex-1 font-semibold rounded-md text-base;
 }
+
+/* disabled styles */
+.payment-button-disabled {
+  @apply relative after:absolute after:content-[''] after:top-0 after:left-0 after:w-full after:h-full;
+}
+
+.payment-form-disabled {
+  @apply payment-button-disabled cursor-not-allowed after:bg-white after:opacity-50 after:z-[5];
+  @apply focus:border-blue-400 focus:outline-none focus:shadow-input-blue-focus;
+}
+
+/* confirm payment checkbox tooltip styles */
+.input-row--checkbox[data-field-name='confirm'] .tooltip {
+  @apply z-[6] px-4 py-1.5 left-0 -mt-12;
+}
+
+.input-row--checkbox[data-field-name='confirm'] .tooltip::before {
+  @apply left-7;
+}
+
+html[dir='rtl'] .input-row--checkbox[data-field-name='confirm'] .tooltip {
+  @apply left-auto right-0;
+}
```

Comment:
Q: Instead of setting `left-n` and then having to override for rtl, can we set `start-n`? (assuming that correctly applies ltr-vs-rtl padding)

```
start-7 | inset-inline-start: 1.75rem; /* 28px */
end-7   | inset-inline-end: 1.75rem; /* 28px */
left-7  | left: 1.75rem; /* 28px */
right-7 | right: 1.75rem; /* 28px */
```

https://tailwindcss.com/docs/top-right-bottom-left#using-logical-properties

Reply:
Added to follow-up ticket and will give it a try there.